src/sidebar/components/logged-out-message.js

```diff
@@ -6,10 +6,18 @@ import { withServices } from '../util/service-context';
 import Button from './button';
 import SvgIcon from '../../shared/components/svg-icon';
 
+/**
+ * @typedef LoggedOutMessageProps
+ * @prop {() => any} onLogin
+ * @prop {(name: string) => any} serviceUrl
```

Comment:
Can you use the `ServiceUrlGetter` type from `src/sidebar/services/service-url.js` here. This will make it easier to keep in sync if it ever changes.